R/data_natalia.R

```diff
@@ -0,0 +1,37 @@
+#' Beto O'Rourke Ignored Cable News - And It Ignored Him
+#'
+#' The raw data behind the story
+#' "Beto O'Rourke Ignored Cable News - And It Ignored Him"
+#' \url{https://fivethirtyeight.com/features/beto-orourke-ignored-cable-news-and-it-ignored-him/}
+#'
+#' @format A data frame with 972 rows representing weeks of cable coverage and 7 variables:
+#' \describe{
+#'   \item{date}{start date for the week of coverage}
+#'   \item{name}{candidate's name}
+#'   \item{matched_clips}{number of 15-second clips in that week that mention the specified candidate}
+#'   \item{all_candidate_clips}{number of 15-second clips in that week that mention any candidates}
+#'   \item{total_clips}{total number of 15-second clips that week across the three networks}
+#'   \item{pct_of_all_candidate_clips}{percentage of clips in which that specific candidate is mentioned out of all clips mentioning any candidate for that week (matched_clips / all_candidate_clips)}
+#'   \item{query}{query used for the GDELT Television API}
+#' }
+#' @source The GDELT Television API \url{https://blog.gdeltproject.org/gdelt-2-0-television-api-debuts/}, which processes the data from the TV News Archive \url{https://archive.org/details/tv}.
+"media_mentions_cable"
+
+
```

Comment:
@rudeboybert do you want these to appear together in the documentation? Perhaps `@rdname` would be appropriate here?

Reply:
@beanumber is there a way I could do this, as each of the two datasets has different variables? I tried doing this, but I can only get the format section to show up in the documentation for the first dataset, as they each need different format/describe documentation.